vyxal/elements.py

```diff
@@ -1163,11 +1163,26 @@ def from_base(lhs, rhs, ctx):
 
 def gen_from_fn(lhs, rhs, ctx):
     """Element Ḟ
-    (fun, lst) -> Generator from a with initial vector b
-    (lst, fun) -> Generator from b with initial vector a
+    (num, num) -> sympy.N(a, b)
```

Comment:
Maybe add a description of what `N` itself does here? (perhaps `evaluate a to b decimal digits, then convert to string`)